### PR TITLE
feat(ui): Added new Clearing State and Attachment Type

### LINKED ISSUE #1390: To support uploading of new automatically generated scan reports to SW360, a new type of clearingState shall be added to sw360
### Description

(from sw360 user) 

To support uploading of new automatically generated scan reports to SW360, a new type of clearingState shall be added to sw360

Also, a corresponding new attachment type should be created probably with a different color code. And attachments with this `ATTACHMENT_TYPE` should not trigger a `CLEARING_STATE` change. (e.g from `NEW` to `REPORT_AVAILABLE`) like how the `CLX` and `CRT` does at present.

### PR DESCRIPTION
**Pull request by JaideepPalit**
> Added new Clearing State and Attachment Type.
> * Which issue is this pull request belonging to and how is it solving it? (*#1390*)
> * Did you add or update any new dependencies that are required for your change?- No

Issue: closes #1390

> Have you implemented any additional tests? - No

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/editAttachments.jspf [hunk 1]
```diff
@@ -180,6 +190,23 @@
             }
         });
 
+        let functionForCheckStatusOfInitialScanType = function (attachmentType, checkStatus) {
+            if(attachmentType == "18") {
```
Comment (ravi110336) [inline frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/editAttachments.jspf:194]: Indentation after if, same for line 203.

**Review by ravi110336 — COMMENTED** [review]
Code looks good to me, please check some indentation comment.

**Comment by ravi110336** [comment]
Tested the PR for a new attachment type and clearing state filter in project and working fine.

File: frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/editAttachments.jspf (hunk 1 above)
Comment (JaideepPalit) [reply frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/editAttachments.jspf]: Fixed

**Comment by JaideepPalit** [comment]
Resolved merge conflict. Incorporated changes

**Comment by ravi110336** [comment]
Hi Jaideep ,

Thanks for incorporating the changes. Tested for the new attachment type for the release, component and Project, when we upload a new attachment and select the new attachment type **Initial Scan Report** and then update the release, component and project. The type is empty in this case and not showing as **ISR**. 

**Please check the below screenshot:** 
image

image

**Comment by JaideepPalit** [comment]
Rebased the branch

**Comment by JaideepPalit** [comment]
Not sure what could be the issue.

I am not able to reproduce the error - 

image

**Comment by ravi110336** [comment]
Hi @JaideepPalit , The issue was in my System, i deploy the branch into my new VM and it is working fine.

image

**Pull request merged**